Easydict/NewApp/View/SettingView/Tabs/View/Genearl/GeneralOtherSetting.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  GeneralOtherSetting.swift
+//  Easydict
+//
+//  Created by Sharker on 2024/1/1.
+//  Copyright © 2024 izual. All rights reserved.
+//
+
+import SwiftUI
+
+@available(macOS 13.0, *)
+extension GeneralTab {
+    struct OtherSettingView: View {
+        var body: some View {
+            Section {
+                HStack {
+                    Text("show_main_window")
+                    Toggle(isOn: $hideMainWindow) {
+                        Text("hide_main_window")
+                    }
+                }
+                HStack {
+                    Text("launch")
+                    Toggle(isOn: $launchAtStartup) {
+                        Text("launch_at_startup")
+                    }
+                }
+                HStack {
+                    Text("menu_bar_icon")
+                    Toggle(isOn: $hideMenuBarIcon) {
+                        Text("hide_menu_bar_icon")
+                    }
+                }
+                HStack {
+                    Text("beta_new_app")
+                    Toggle(isOn: $enableBetaNewApp) {
+                        Text("enable_beta_new_app")
+                    }
+                }
+            } header: {
+                Text("other")
+            }
+        }
+
+        @AppStorage(kHideMainWindowKey) private var hideMainWindow = false
```

Comment:
We can replace this with `@Default(.hideMainWindow) private var hideMainWindow`.

Reply:
okay I  got it